### ISSUE TITLE
There is no softmax layer in the paper

### ISSUE THREAD

**Issue by Atcold** [aliasvishnu/Capsule-Networks-Notebook-MNIST#1]
This line is simply wrong
```python
classes = F.softmax(classes)
```

**Comment by aliasvishnu** [comment]
Softmax is not used in the paper, but the sum of lengths > 1, softmax would be needed to use CrossEntropyLoss.

**Comment by Atcold** [comment]
Yeah, I just saw that you're not trying to reproduce the paper, but having your own version.

**Comment by aliasvishnu** [comment]
Added, thanks for your comments.
